gsys/src/lib.rs

```diff
@@ -331,6 +331,13 @@ impl TwoHashesWithValue {
 
 #[allow(improper_ctypes)]
 extern "C" {
+    /// Infallible `gr_exec_settings` get syscall.
+    ///
+    /// Arguments type:
+    /// - `settings`: `mut ptr` for buffer to store requested version of settings.
+    /// - `version`: `u32` defining version of settings to get.
+    pub fn gr_exec_settings(settings: *mut BufferStart, version: u32);
```

Comment:
```suggestion
    pub fn gr_exec_settings(version: u32, settings: *mut BufferStart);
```

Reply:
Any explanation for the requested change?

Reply:
Consistency with other sys-calls where input arguments are first, output ones are second